geometry_msgs/Point start_pnt
std_msgs/Float32 start_vel
std_msgs/Float32 start_acc
std_msgs/Float32 start_yaw
geometry_msgs/Point goal_pnt
std_msgs/Float32 goal_vel
std_msgs/Float32 goal_acc
std_msgs/Float32 goal_yaw
std_msgs/Float32 T
std_msgs/Float32 dt
---
std_msgs/Float32[] x_coeff
std_msgs/Float32[] y_coeff
nav_msgs/Path path
geometry_msgs/Twist[] vels
geometry_msgs/Accel[] accs
